.github/workflows/cancel_duplicate_workflow_runs.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   cancel-duplicate-workflow-runs:
+    if: github.event.workflow_run.event == 'pull_request'
```

Comment:
@dongjoon-hyun, I referred to https://securitylab.github.com/research/github-actions-preventing-pwn-requests and found this way is slightly better because it's at least easier to understand. Likewise, I tested it in my fork, and updated PR description.